systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -218,6 +292,20 @@ void testCruiseControlChangesFromRebalancingtoProposalReadyWhenSpecUpdated() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Test verifying that Cruise Control cannot be deployed with a Kafka cluster that has only one broker and ensuring that increasing the broker count resolves the configuration error."),
+        steps = {
+            @Step(value = "Initialize test storage and set up the error message", expected = "Test storage initialized and error message set"),
+            @Step(value = "Create and wait for resources with node pools.", expected = "Resources are created successfully."),
```

Comment:
I saw it in the other test cases' steps as well and it looks a bit strange to me -> there is not much wait for the creation (there is not much time we are waiting for the NodePools to be created). Also the `with` looks a bit strange to me, so maybe something like this:
```suggestion
            @Step(value = "Create broker and controller KafkaNodePools.", expected = "Both KafkaNodePools are successfully created"),
```

I would also use the same step for all other testcases as well, making the documentation more consistent. Surely if there is some change in the configuration or if they are using persistent storage instead of ephemeral or whatever, you can change it. But I would not create variation of the same sentence in every test case.

Reply:
Sure, make sense updated.